Packs/PagerDuty/Integrations/PagerDuty/PageDuty_test.py

```diff
@@ -41,3 +42,278 @@ def test_get_incidents(requests_mock, mocker):
     )
     res = get_incidents_command()
     assert '| Documentation: • |' in res['HumanReadable']
+
+
+def test_add_responders(requests_mock, mocker):
+    """
+    Test sending request to a responder
+    """
+    mocker.patch.object(
+        demisto,
+        'params',
+        return_value={
+            'APIKey': 'API_KEY',
+            'ServiceKey': 'SERVICE_KEY',
+            'FetchInterval': 'FETCH_INTERVAL',
+            'DefaultRequestor': 'P09TT3C'
+        }
+    )
+    mocker.patch.object(
+        demisto,
+        'args',
+        return_value={
+            "incident_id": "PXP12GZ",
+            "message": "Please help with issue - join bridge at +1(234)-567-8910",
+            "user_requests": "P09TT3C,PAIXXX"
+        }
+    )
+    requests_mock.post(
+        'https://api.pagerduty.com/incidents/PXP12GZ/responder_requests',
+        json={
```

Comment:
please put all the json mock response in a file, it would make the unit test more readable